packages/fluentui/react-northstar/src/components/Tree/Tree.tsx

```diff
@@ -73,6 +93,12 @@ export interface TreeProps extends UIComponentProps, ChildrenComponentProps {
    * @returns The render prop result.
    */
   renderedItems?: (renderedItems: React.ReactElement[]) => React.ReactNode;
+
+  /** Whether or not tree items are selectable. */
+  selectable?: boolean;
+
+  /** Allows user to pass a custom component to replace the current checkbox */
+  customSelectIndicator?: (props: CustomSelectIndicatorProps) => JSX.Element;
```

Comment:
I was thinking that we agreed to omit this one?

Reply:
Not this one. we omitted the prop related to the position of the indicator. This one is to allow the user to pass a custom checkbox to replace the default one.

Reply:
Completely removed